src/css/ArticleData.css

```diff
@@ -1,4 +1,4 @@
-div>div>div>div>img {
+div > div > div > div > img {
```

Comment:
This is actually bad. Because this will prevent any changes in markup. Please just use class.